src/subscription_manager/gui/registergui.py

```diff
@@ -814,6 +873,16 @@ def create_wizard_widget(self, backend, facts, reg_info, parent_window):
         return autobind_widget
 
 
+class myDec(object):
```

Comment:
Nah, I just wanted to log when each subclass's pre and pre_done were called to see if they overlapped. This was the result of my lack of desire to copy paste a log debug statement into all subclasses :smile:

It will be removed along with the rest of the debug bits.